python/travis/run_python_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#
+#  Copyright 2015-2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+set -ev
+
+source activate flint
+
+PYSPARK_PYTHON=python3 spark/bin/spark-submit --master=local --jars ../target/scala-2.11/flint-assembly-0.2.0-SNAPSHOT.jar --py-files ../target/scala-2.11/flint-assembly-0.2.0-SNAPSHOT.jar test/test_dataframe.py
```

Comment:
https://github.com/holdenk/spark-testing-base this seems to download the spark dependency for you, not sure how pyspark works.

`findspark` allows testing not using spark-submit, but still needs SPARK_HOME and therefore, a spark distribution jar.

I think requiring a spark distribution jar is ok.